webdav/cc_domain_controller.py

```diff
@@ -84,39 +99,75 @@ def basic_auth_user(self, realm, user_name, password, environ):
     True if user is authorized
     """
     logging.debug('Start basic magicauth user')
-    username = environ['REMOTE_USER'] # example avalingot@CCOMPTES.FR
-    username = username.split('@', 1)[0]
 
-    # We know that the following user exists in the LDAP
-    # We now check the realm
-    try:
-      logging.info(f'Basic magicauth: LDAP Sever (username: {username})')
+    def get_samaccount_from_env(environ):
+      """
+      The format for REMOTE_USER is samaccount@CCOMPTES.FR.
+      Example : for Caroline Elbourki : celbourki@CCOMPTES.fr
+      The REMOTE_USER environment variable was set by the webserver.
```

Comment:
You coud use a normed way to document you function for example 
```suggestion
      The REMOTE_USER environment variable was set by the webserver.
      param: environ: Environ Variable from apache and WSGI context
      return: SamAccount Name as ecomment
```